### PR TITLE
Document Dev Summit's Facilitator role

### PR DESCRIPTION
**Pull request by ArthurSens**
Following what was discussed in yesterday's Dev summit, here's the PR documenting the Facilitator role.

### CODE REVIEW

File: content/community.html [hunk 1]
```diff
@@ -157,6 +157,35 @@ <h1>Developer summits</h1>
...
+    <ul>
+      <li>Two or three days before the meeting, send reminders in our public community channels inviting people to add Agenda Topics, and Prometheus Team members and maintainers to vote on topics they'd like to discuss.</li>
+      <li>One day before the meeting, reach out to "Topic owners" who received the most votes to make sure they'll make it to the summit.</li>
+    </ul>
+    <h3>During the summit</h3>
+    <p>
+      During the summit, the Facilitator is here to make sure the meeting runs smoothly, and that consensus is reached when needed. We suggest the following tasks:
+    </p>
+    <ul>
+      <li>Start the meeting on time.</li>
```
Comment (bwplotka) [inline content/community.html:177]:
```suggestion
      <li>Start the meeting on time. Use @prometheus account for the admin meeting permissions.</li>
```

File: content/community.html [hunk 2]
```diff
@@ -157,6 +157,35 @@ <h1>Developer summits</h1>
...
+      <li>Start the recording and mention that the Code of Conduct applies.</li>
+      <li>Select topics to be discussed based on votes and who is currently present in the meeting.</li>
+      <li>Transcribe the meeting notes in the shared document.</li>
```
Comment (bwplotka) [inline content/community.html:180]:
```suggestion
      <li>Take notes or find volunteer for taking notes in the shared document.</li>
```

**Review by bwplotka — APPROVED** [review]
Thanks! Epic work, suggested some minor additions. LGTM

File: content/community.html [hunk 3]
```diff
@@ -157,6 +157,35 @@ <h1>Developer summits</h1>
...
+    <ul>
+      <li>Two or three days before the meeting, send reminders in our public community channels inviting people to add Agenda Topics, and Prometheus Team members and maintainers to vote on topics they'd like to discuss.</li>
+      <li>One day before the meeting, reach out to "Topic owners" who received the most votes to make sure they'll make it to the summit.</li>
+    </ul>
+    <h3>During the summit</h3>
+    <p>
+      During the summit, the Facilitator is here to make sure the meeting runs smoothly, and that consensus is reached when needed. We suggest the following tasks:
+    </p>
+    <ul>
+      <li>Start the meeting on time. Use @prometheus account for the admin meeting permissions.</li>
```
Comment (codesome) [inline content/community.html:177]:
```suggestion
      <li>Start the meeting on time. Use @prometheus.io account for the admin meeting permissions.</li>
```

**Review by codesome — APPROVED** [review]

**Pull request merged**